.github/workflows/deployment.yml

```diff
@@ -0,0 +1,62 @@
+# Copied from https://github.com/notlmn/browser-extension-template/blob/master/.github/workflows/deployment.yml
+name: Deployment
+
+on:
+  workflow_dispatch:
+  # You can manually trigger a deployment on GitHub.com
+  # https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/
+  schedule:
+    - cron: '31 13 * * 3'
+
+jobs:
+  Build:
+    outputs:
+      created: ${{ steps.daily-version.outputs.created }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 20
+      - run: npm install
+      - run: npm test # This includes the build
+      - uses: fregante/daily-version-action@v1
+        name: Create tag if necessary
+        id: daily-version
+      - uses: notlmn/release-with-changelog@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          template: '{commits}'
+          exclude: '^Meta|^Document|^Readme|^Lint'
+      - name: Update extension’s meta
+        env:
+          VER: ${{ steps.daily-version.outputs.version }}
+        run: |
+          echo https://github.com/$GITHUB_REPOSITORY/tree/$VER > distribution/SOURCE_URL.txt
+          npm run version
```

Comment:
Ensure that the npm scripts are also up to date. Maybe they can be dropped or must match this workflow. I never set this up without also changing the scripts.